.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,21 @@
+# Release Description
+<!--- Summarize what is being released.  -->
+
+## Milestone
+<!--- Link to the milestone for the release. ---> 
+<!--- Make sure all relevant issues/PRs are included on the linked pages. --->
+Milestone: 
+
+# Release Checklist
+<!--- Fill out the following Release checklist -->
+
+- [ ] Version number has been updated
+- [ ] NEWS.md has been updated
+- [ ] README.md has been updated (if applicable)
+- [ ] Vignettes have been updated (if applicable)
+- [ ] Ensure all unit tests are passing
+- [ ] All QC Checklists in this PR are complete
```

Comment:
Do we have a QC Checklist?  I'm intrigued?!?

Reply:
We don't, this fell through my copy and paste check